File: 프로그래머스/unrated/181837. 커피 심부름/커피 심부름.java
class Solution {
    public int solution(String[] order) {
        String s1 = "latte";
        int ame = 0;
        int latte = 0;
        for(String s : order){
            if(s.contains(s1)) latte++;
            else ame++;
        }
        int ret = 0;
        ret = ame*4500 + latte*5000;
        return ret;
    }
}

File: 프로그래머스/0/181837. 커피 심부름/커피 심부름.py
def solution(orders):
    ret = 0
    for order in orders:
        if order.find("cafelatte") != -1:
            ret += 5000
        else: ret += 4500
    return ret
    
# 메뉴만 => 아이스, 아무거나 => 아메리카노